net/lighttpd/Makefile

```diff
@@ -8,12 +8,12 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=lighttpd
-PKG_VERSION:=1.4.55
-PKG_RELEASE:=3
+PKG_VERSION:=1.4.56
```

Comment:
for testing purposes you can set this to 1.4.56~rc2

Reply:
For testing purposes, I am using git-src symlink. :)

Reply:
Right but the CI cannot build this.

Reply:
ackn.  will update with lighttpd-1.4.56~rc3 after I push rc3 in the next day or so.  There are some meson.build fixes that have been committed, but are not part of rc2.